### PR TITLE
sunpy 2020

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: _projects/2020/sunpy/sunspotter.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+name: Space Weather forecasting using linear algebra
```
Comment (bsipocz) [inline _projects/2020/sunpy/sunspotter.md]: 🥇

**Comment by bsipocz** [comment]
@nabobalis - let me know when to hit the button.

**Comment by nabobalis** [comment]
Will do. I am hoping to get speedy reviews.

File: _projects/2020/sunpy/fido.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+issues:
+- https://github.com/sunpy/sunpy/pulls?q=is%3Apr+label%3A%22Needs+Adoption%22+is%3Aclosed+label%3Aunidown
+- https://github.com/sunpy/sunpy/issues/3336
+- https://github.com/sunpy/sunpy/issues/3735
+- https://github.com/sunpy/sunpy/issues/3733
+- https://github.com/sunpy/sunpy/issues/3734
+- https://github.com/sunpy/sunpy/issues/2744
+- https://github.com/sunpy/sunpy/issues/2631
+mentors:
+ - cadair
```
Comment (bsipocz) [inline _projects/2020/sunpy/fido.md]: The rendering might be case sensitive?

Comment (nabobalis) [reply _projects/2020/sunpy/fido.md]: Good point. I will fix that.

**Comment by bsipocz** [comment]
I'm sorry @Cadair. Deadline is close, and I rather have not perfect ideas on the webpage when we submit than perfect ones a bit later. You can always update them in a follow-up.

**Pull request merged**